indexer/ftypes_matcher.hpp

```diff
@@ -238,7 +238,7 @@ class IsPlaceChecker : public BaseChecker
 
 class IsBridgeChecker : public BaseChecker
 {
-  virtual bool IsMatched(uint32_t type) const override;
+  bool IsMatched(uint32_t type) const override;
```

Comment:
Why?

Reply:
it is a bad idea to have a class with virtual methods but with no virtual destructor. ideally, besides removing virtual specified, this class should also be marked as final